src/MICore/Transports/StreamTransport.cs

```diff
@@ -15,6 +15,8 @@ namespace MICore
 {
     public abstract class StreamTransport : ITransport
     {
+        protected const int STREAM_BUFFER_SIZE = 65536;
```

Comment:
There shouldn't be a need to change anything other than pipe transport, so please move your comment there